requirements.txt

```diff
@@ -4,3 +4,4 @@ versioneer
 dipy==1.0.0
 svgutils
 nipype==1.2.0
+sentry_sdk==0.13.1
```

Comment:
I think requirements.txt might not be needed anymore since these are setup.cfg. Not sure, though